roles/puma/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Install puma app service
+
+- name: copy puma files
+  template:
+    src={{ item.src }}
+    dest={{ item.dest }}
+    mode={{ item.mode }}
+    owner={{ item.owner }}
+    group={{ item.group }}
```

Comment:
In Ansible we can use `src={{ item.src }}` or  `src: {{ item.src }}`. I actually prefer the latter. It's more consistent withing the file. What do you think?

Reply:
That makes sense, if I have time I will update it.